projects/sandbox/src/app/wizard/wizard.component.ts

```diff
@@ -1,19 +1,20 @@
-import { Component, Input } from '@angular/core';
+import { Component } from '@angular/core';
 import { FormBuilder, FormControl, FormGroup, Validators } from '@angular/forms';
 import { HttpErrorResponse } from '@angular/common/http';
 import { merge, Observable, timer } from 'rxjs';
-import { DataPollingComponent } from '@shared';
+import { DataPollingComponent } from 'shared';
 import { apiSettings } from '../../environments/apisettings';
 import {
   DatasetInfo,
   DatasetInfoStatus,
+  FieldOption,
+  FixedLengthArray,
   SubmissionResponseData,
-  WizardField,
   WizardStep,
   WizardStepType
 } from '../_models';
 import { SandboxService } from '../_services';
-import { ProtocolType } from '@shared';
+import { ProtocolType } from 'shared';
 
 export interface ValidatorArrayHash {
```

Comment:
According to my IDE this is now unused.

Reply:
It is used.  You probably need to re-build the shared library.